core/src/test/scala/org/apache/spark/rpc/RpcEnvSuite.scala

```diff
@@ -658,7 +657,7 @@ abstract class RpcEnvSuite extends SparkFunSuite with BeforeAndAfterAll {
       })
       val rpcEndpointRef = remoteEnv.setupEndpointRef(localEnv.address, "send-authentication")
       rpcEndpointRef.send("hello")
-      eventually(timeout(5 seconds), interval(10 millis)) {
+      eventually(timeout(5.seconds), interval(10.millis)) {
```

Comment:
`10.millis` => `10.milliseconds`

There are a few other places where this pattern is repeated:
```
(pr/24314) $ git diff 017919b636  | grep "^\+.*millis[^e]"
+    eventually(timeout(1.minute), interval(10.millis)) {
+        rpcEndpointRef.askSync[String]("hello", new RpcTimeout(1.millis, shortProp))
+      eventually(timeout(5.seconds), interval(10.millis)) {
+    eventually(timeout(1.minute), interval(200.millis)) {
+      Await.ready(f, 50.millis)
+      eventually(timeout(20.seconds), interval(batchDuration.milliseconds.millis)) {
+      eventually(timeout(10.seconds), interval(10.millis)) {
```

Reply:
Of course this is just a Nit as both correct (but from your previous changes I have the feeling milliseconds is prefered).